scripts/bulk_capture.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Help message
+function show_help {
+    echo "Usage: $0 [--help]"
+    echo ""
+    echo "  --help       Print this message"
+    exit 1
+}
+
+# Check if --help is passed
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --help) show_help ;;
+        *) echo "Unknown parameter passed: $1"; show_help ;;
+    esac
+    shift
+done
+
+# File containing the ranges
+range_file="./range.csv"
+
+# Generate a unique temporary directory name
+uuid=$(uuidgen)
+remote_dir="/tmp/replays-$uuid"
+
+# Check if the range file exists
+if [ ! -f "$range_file" ]; then
+    echo "Error: range file $range_file does not exist"
+    exit 1
+fi
+
+# Initialize the full command
+full_cmd="mkdir -p $remote_dir; "
+
+# Variables to track the very first and very last ranges
+very_first=""
+very_last=""
+
+# Read the range file and process each range
+while IFS= read -r line || [[ -n "$line" ]]; do
+    # Debug: Show the current line being processed
+    echo "Debug: Current line: '$line'"
+
+    # Skip empty lines
+    if [ -z "$line" ]; then
+        echo "Debug: Skipping empty line"
+        continue
+    fi
+
+    # Extract start and end from the line
+    start=$(echo "$line" | cut -d'-' -f1)
+    end=$(echo "$line" | cut -d'-' -f2)
+
+    # Debug: Show extracted start and end values
+    echo "Debug: Extracted start: '$start', end: '$end'"
+
+    # Check if start and end are properly extracted
+    if [ -z "$start" ] || [ -z "$end" ]; then
+        echo "Debug: Skipping invalid line: '$line'"
+        continue
+    fi
+
+    # Track the very first and very last ranges
```

Comment:
Given the names below wouldn't it make sense to track the smallest `$start` and the largest `$end` (if both are well-formed and in the correct order, of course) ?

Reply:
Agree. This way the file name will be more  meaningful. Will fix this.

Reply:
updated